eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -824,7 +825,7 @@ invalid_hereafter = "invalid_hereafter" =:~ grp [5, a slot_no]
 multiasset :: IsType0 a => a -> GRuleCall
 multiasset = binding $ \x ->
   "multiasset"
-    =:= mp [1 <+ asKey policy_id ==> mp [1 <+ asKey asset_name ==> x]]
+    =:= mp [0 <+ asKey policy_id ==> mp [1 <+ asKey asset_name ==> x]]
```

Comment:
This is also not accurate: 
https://github.com/IntersectMBO/cardano-ledger/blob/ca8d451bbce11dde3b68e99782c79f9b4c1dfca5/eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs#L328

It will never happen in Conway that an empty multi asset is decoded.
```suggestion
    =:= mp [1 <+ asKey policy_id ==> mp [1 <+ asKey asset_name ==> x]]
```
See this two:
* Empty outer map is not allowed in mint field: https://github.com/IntersectMBO/cardano-ledger/blob/ca8d451bbce11dde3b68e99782c79f9b4c1dfca5/eras/conway/impl/src/Cardano/Ledger/Conway/TxBody.hs#L188
* In `Value` when multi-asset is present it also will have to be non-empty: https://github.com/IntersectMBO/cardano-ledger/blob/ca8d451bbce11dde3b68e99782c79f9b4c1dfca5/eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs#L294-L298